FastSimulation/SimplifiedGeometryPropagator/interface/ParticleManager.h

```diff
@@ -140,4 +141,9 @@ namespace fastsim {
   };
 }  // namespace fastsim
 
+inline bool isExotic(int pdgid_) {
```

Comment:
Please add a comment explaining the function and the numbers (eg. `3000022`).

Reply:
added comments here from Vladimir